app/views/users/_connections.html.erb

```diff
@@ -0,0 +1,9 @@
+<div class="row">
+  <div class="col-md-6 offset-3">
+    <div class="list-group">
+      <% connections.each do |connection| %>
+        <%= link_to "#{connection.username}", user_path(connection.username), class: "list-group-item list-group-item-action" %>
+      <% end %>
+    </div>
+  </div>
+</div>
```

Comment:
That's interesting how you make this '_connection.html.erb' partial work both for followers and for following.